sites/all/modules/reol_lazy_load_images/reol_lazy_load_images.module

```diff
@@ -140,7 +143,6 @@ function theme_reol_base_image($variables) {
   else {
     $attributes['src'] = file_create_url($variables['path']);
   }
-
```

Comment:
Be careful with removing blank lines (and other stuff) that's not part of the actual code changes that you're making. A pull request should only contain changes relevant to the actual issue.